docs/sources/datasources/loki/_index.md

```diff
@@ -15,81 +15,36 @@ weight: 800
 
 # Loki data source
 
-Grafana ships with built-in support for [Loki](/docs/loki/latest/), an open-source log aggregation system by Grafana Labs.
-This topic explains configuring and querying specific to the Loki data source.
+Grafana Loki is a set of components that can be composed into a fully featured logging stack.
```

Comment:
Personally, I don't understand this sentence and it made more confused than previously used `an open-source log aggregation system`.